laser_tilt_controller:
    controller:
        package: ax12_driver_core
        module: joint_position_controller
        type: JointPositionControllerAX12
    joint_name: hokuyo_tilt_joint
    joint_speed: 1.17
    motor:
        id: 7                       #9
        init: 817
        min: 817
        max: 663
